src/nodedef.cpp

```diff
@@ -618,6 +621,11 @@ void ContentFeatures::deSerialize(std::istream &is)
 		if (is.eof())
 			throw SerializationError("");
 		alpha = static_cast<enum AlphaMode>(tmp);
+
+		u8 tmp_move_resistance = readU8(is);
+		if (is.eof())
+			throw SerializationError("");
```

Comment:
What about `move_resistance = liquid_viscosity` instead of `throw SerializationError("");`? That should fix the comment below.